lib/sidekiq_unique_jobs/middleware/client/strategies/unique.rb

```diff
@@ -75,7 +75,7 @@ def old_unique_for?
 
           def new_unique_for?
             connection do |conn|
-              return conn.set(payload_hash, item['at'] ? 2 : 1, nx: true, ex: expires_at)
+              return conn.set(payload_hash, item['jid'], nx: true, ex: expires_at)
```

Comment:
> (Since SETNX will always fail if any value is set.) Was the idea to let simultaneous scheduling and queuing of jobs? I was under the impression this was deprecated but lacking code-comments it's hard to tell the intent? 
Those are equivalent since it is being used with watch or at least that was the intention.